src/app/page.tsx

```
@@ -1,116 +1,380 @@
         />
+        <div className="absolute inset-0 bg-gradient-to-r from-[#0D3234]/75 via-[#0D3234]/35 to-transparent" />
+      </div>
+      <div className="relative z-10 max-w-6xl mx-auto px-6 py-10">
+        <div className="max-w-lg">
+          <span className="inline-flex items-center gap-2 rounded-full bg-white/20 backdrop-blur-sm px-3 py-1 text-xs font-semibold text-white mb-3">
+            <MapPin className="size-3" />
+            Supporting Mothers Across Rwanda
+          </span>
+          <h1 className="text-4xl sm:text-5xl font-bold text-white leading-tight">
+            Empowering Every{" "}
+            <span className="text-[#7ECFD4]">Mother's Journey</span>{" "}
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)